examples_gbs/run_tutorial_points.py

```diff
@@ -1,65 +1,87 @@
-"""
+# pylint: disable=wrong-import-position,wrong-import-order,ungrouped-imports,invalid-name
+r"""
 Point Process Tutorial
 ======================
 
 This tutorial shows how to generate GBS point process samples and use them to detect outlier
 points in a data set. Point processes are models for generating random point patterns and GBS
 devices can be programmed to operate as special types of point processes that generate clustered
-point patterns. GBS point processes belong to a class of point processes in which the probability
-of generating a specific pattern of points depends on matrix functions of a kernel matrix that
-describes the similarity between the points. Matrix functions that appear in GBS point processes
-are typically permanents and hafnians. In this tutorial we use the permanental point process in which
-the probability of observing a pattern of points depends on the permanent of their corresponding
-kernel matrix. 
+random point patterns.
+
+In GBS point processes, the probability of generating a specific pattern of points depends on
+matrix functions of a kernel matrix that describes the similarity between the points. Matrix
+functions that appear in GBS point processes are typically permanents and hafnians. In this
+tutorial, we use the permanental point process in which the probability of observing a pattern of
+points :math:`S` depends on the permanent of their corresponding kernel matrix :math:`K_S` as:
```

Comment:
Might be good to provide a link to more information on the permanent, e.g. its wikipedia page